cypress/integration/pages/mediaAssetPage/tests.js

```diff
@@ -77,17 +77,23 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
     it('should have href that matches assetURI for 1st related content link', () => {
       cy.request(`${Cypress.env('currentPath')}.json`).then(({ body }) => {
         const numRelatedContentGroups = body.relatedContent.groups.length;
-
-        if (numRelatedContentGroups > 0) {
+        const assetUriString = body.metadata.locators.assetUri;
+        if (
+          numRelatedContentGroups > 0 &&
+          assetUriString.split('/').length <= 5
```

Comment:
For understandability - could `assetUriString.split('/').length <= 5` be assigned to an 'is not legacy asset' variable (or indeed an 'is legacy asset' variable as I think is the pattern within the rest of the tests?) and the variable used in this condition instead? 

Currently the only way of knowing what this if () statement is capturing is via the cy.log statement in the else.

Reply:
Karina has outlined the explicit solution to this here https://github.com/bbc/simorgh/pull/5768#pullrequestreview-369475560